src/cmdc_tools/datasets/official/HHS/__init__.py

```diff
@@ -0,0 +1,93 @@
+import pandas as pd
+
+import us
+
+from ..base import ArcGIS, CountyData
+from ... import DatasetBaseNoDate
+
+
+class HHS(DatasetBaseNoDate, ArcGIS):
+    """
+    This scraper retrieves the hospital data provided by HHS at:
+
+      https://protect-public.hhs.gov/pages/hospital-capacity
+
+    It includes information on hospital capacity, hospital beds in use,
+    hospital beds in use by covid patients, icu capacity, icu beds in
+    use, and icu beds in use by covid patients
+    """
+
+    ARCGIS_ID = "qWZ7BaZXaP5isnfT"
+    table_name = "us_covid"
+    pk = '("vintage", "dt", "fips", "variable_id")'
+    has_fips = True
+    state_fips = 0  # Using 0 to denote that this is a national database
+    source = "https://protect-public.hhs.gov/pages/hospital-capacity"
+    provider = "HHS"
+
+    def _get_percent_reporting(self):
+        # TODO: See what pattern for the variable,
```

Comment:
```
In [22]: bar = self.get_all_sheet_to_df( 
    ...:                 "Percentage_of_Facilities_Reporting_by_State", 0, 5 
    ...:             )                                                                                                                                                            

In [23]: list(bar)                                                                                                                                                                
Out[23]: 
['OBJECTID',
 'state_name',
 'state_fips',
 'state_abbr',
 'hhs_region',
 'fac_reporting_20200720',
 'definitive_fac',
 'pct_fac_reporting_def',
 'Shape__Area',
 'Shape__Length']

In [41]: (np.round(bar.eval("(definitive_fac *  pct_fac_reporting_def / 100)")) - bar["fac_reporting_20200720"]) .head(10)                                                        
Out[41]: 
0    0.0
1    0.0
2    0.0
3    0.0
4    0.0
5    0.0
6    0.0
7    0.0
8    0.0
9    0.0
dtype: float64
```

Do we want to compute the `number_facilities_reporting` as `definitive_fac * pct_fac_reporting_def/100` to avoid this problem?

Reply:
Good idea.